pkg/proxy/iptables/proxier.go

```diff
@@ -1497,8 +1497,8 @@ func (proxier *Proxier) syncProxyRules(reason syncReason) {
 	proxier.iptablesLines.Write(proxier.natChains.Bytes())
 	proxier.iptablesLines.Write(proxier.natRules.Bytes())
 
-	if glog.V(4) {
```

Comment:
This is extremely expensive (in my tests in just 100-node cluster, this line is responsible for ~10% of whole cpu usage and ~20% of all memory allocations and GC).

I think that we can just bump the verbosity in tests in case of problems.